composer.json

```diff
@@ -59,6 +59,18 @@
             "locations": {
                 "web-root": "./build"
             }
+        },
+        "patches": {
```

Comment:
do we really need this in our project?

Reply:
The patches are related to the comment above.
Regarding the PHP 8.2 it can be updated, but there is a bug between that version, Drupal and Xdebug here is the issue: https://www.drupal.org/project/drupal/issues/3405976

Reply:
Removed 9 as core requirement

Reply:
Yeah let's stay to 8.1 for local development, that issue is nasty and blocks installation.